### ISSUE TITLE
Make PR's into RE:Minecraft to make reconnecting better instead of Reflecting into it

### ISSUE THREAD

**Issue by EmotionalLove** [TheSocietyProject/ReconnectionManager#2]
I think it would be a lot more useful for all RE:Minecraft plugin developers if anything RE:Minecraft enchancing was a pull request rather than a seperate thingy

**Comment by TheSocietyProject** [comment]
I agree with you. Therefor I build the apis in ReMc now. I rely on you though to accept the prs or at least tell me what I have to change so you can add it
